gatsby-theme-emulsify/src/components/Molecules/Menus/MainMenu.component.js

```diff
@@ -76,7 +76,9 @@ export default class MainMenu extends Component {
         }}
       >
         {Object.keys(groupedMenuItems).map((parentKey, parentIndex) => {
-          const parentName = parentKey.split("__").pop();
+          const parentName = parentKey.split("-").pop();
+          const parentNameSplit = parentName.split("_").pop();
+          const parentNameFinal = parentNameSplit.split("__").pop();
```

Comment:
I actually wonder if this is even necessary. I think the line above would already account for this since it splits after the last underscore

Reply:
good call